extensions/aws/s3/provision/src/test/java/org/eclipse/dataspaceconnector/provision/aws/s3/S3StatusCheckerTest.java

```diff
@@ -36,9 +37,6 @@
 
 @IntegrationTest
 class S3StatusCheckerTest extends AbstractS3Test {
```

Comment:
Since the `S3StatusChecker` changed its implementation by moving over to a for-loop, all tests here will *fail* because an `emptyList()` is passed. See my comment in the `S3StatusChecker.java` for reference.

The same is true for the Azure variant of the Checker and -test.

Reply:
Should be fixed in the patch